sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeSentimentOptions.cs

```diff
@@ -23,9 +23,9 @@ internal AnalyzeSentimentOptions(TextAnalyticsRequestOptions options)
         }
 
         /// <summary>
-        /// If set to true, response will contain Opinion Mining sentiment analysis results.
-        /// Only available for Text Analytics Service version v3.1-preview.1 and above.
+        /// Specifies the types of analysis to apply, like for example,
+        /// Opinion mining.
```

Comment:
super nit: It might be good to treat this comment more like a list, i.e. these are the analysis types we support. This way it'll be cleaner to include which API versions the analysis is available for